<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonRefresh">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Х</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelContent">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="print">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Распечатать список</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="list">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Lucida Console</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushAdd2">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushEdit">
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineSelect">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Найти...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="buttonSaller">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Продать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonOrder">
          <property name="text">
           <string>Заказать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCustomers">
          <property name="text">
           <string>Клиенты</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRecive">
          <property name="text">
           <string>Прием товара</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="statistica">
          <property name="text">
           <string>Статистика</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonOptions">
          <property name="text">
           <string>Настройки</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionOpen_data">
   <property name="text">
    <string>Open data..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
